website/public/favicon/browserconfig.xml

```diff
@@ -2,7 +2,7 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="/favicon/mstile-150x150.png"/>
+            <square150x150logo src="/mstile-150x150.png"/>
```

Comment:
The location of this resource is https://marquee.stateful.com/favicon/mstile-150x150.png